grunz/tests/test_file_utils.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+import tempfile
+import os
+import time
+import hashlib
+from pathlib import Path
+from filecmp import cmp
+
+from grunz.file_utils.file_utils import FileUtils
+
+
+class TestFileUtils(unittest.TestCase):
+
+    def setUp(self):
+        # Creates a temporary directory for test files
+        self.test_dir = tempfile.TemporaryDirectory()
+        self.file_utils = FileUtils(Path(self.test_dir.name))
+
+    def tearDown(self):
+        # Cleans up the temporary directory and test files
```

Comment:
In this case I think given that `tearDown` is a commoon practice, we can omit this comment, how do you see it?

Reply:
Again, agreed, have removed.